app/models/status_edit.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: status_edits
+#
+#  id                        :bigint(8)        not null, primary key
+#  status_id                 :bigint(8)        not null
+#  account_id                :bigint(8)        not null
+#  text                      :text             default(""), not null
+#  spoiler_text              :text             default(""), not null
+#  media_attachments_changed :boolean          default(FALSE), not null
+#  created_at                :datetime         not null
+#  updated_at                :datetime         not null
+#
+
+class StatusEdit < ApplicationRecord
+  belongs_to :status
+  belongs_to :account, optional: true
```

Comment:
Because the foreign key is `on_delete: :nullify`. This mirrors how it's done for `action_taken_by_account_id` on `reports`. My idea was that if a status was updated through the admin UI, we would record that internally just in case. If that's not necessary, the `account_id` column could be skipped altogether.

Reply:
Oh you mean `account` might not be `status.account`? Makes sense to me, but we could add it once we add the ability to do that. I'm fine with it either way.